e2e/test/DeviceClientLifecycleTests.cs

```diff
@@ -30,56 +30,18 @@ public DeviceClientLifecycleTests()
         #region ReceiveAsyncAfterDispose
 
         [TestMethod]
-        public async Task DCLC_ReceiveAsyncAfterDispose_Sasl_Amqp()
+        public async Task DeviceClientLifeCycle_ReceiveAsyncAfterDispose_Sasl_Amqp()
```

Comment:
We're still missing tests for what we expected to happen in 1.18 for MQTT and how this same behavior verified in the latest release.

Reply:
I added the tests again, and will mark them as disabled as we talked.
MQTT tests would fail due to this issue, which I just filed under DotNetty: https://github.com/Azure/DotNetty/issues/502